{{cookiecutter.project_shortname}}/{{cookiecutter.package_name}}/records/api.py

```diff
@@ -0,0 +1,16 @@
+{% include 'misc/header.py' %}
+"""Invenio-Records-Files API."""
+
+from __future__ import absolute_import, print_function
+
+from invenio_records_files.api import Record, FilesMixin
+from invenio_db import db
+from invenio_files_rest.models import Bucket
+
+
+class RecordWithBucket(Record, FilesMixin):
+
+    def _create_bucket(self):
+        bucket = Bucket.create()
+        db.session.commit()
```

Comment:
Isn't that a bit "aggressive"? e.g. accessing the `record.files` property would basically trigger a database transaction commit